examples/pdc_global_component_contact.rb

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+
+require './lib/pdc'
+
+def main
+  PDC.configure do |config|
+    config.site = 'https://pdc.engineering.redhat.com'
+    config.log_level = :debug
+    config.disable_caching = true
+  end
+
+  comp_contact = PDC::V1::GlobalComponentContact.where(
+    component: 'vlgothic-fonts'
+  ).find_one!
+  puts comp_contact.contact
+end
+
+main if __FILE__ == $PROGRAM_NAME
```

Comment:
There is a error occurred on travis when execute "bundle exec pronto run -f github_pr -c origin/${TRAVIS_BRANCH}".

> /home/travis/.rvm/gems/ruby-2.2.2/gems/octokit-4.7.0/lib/octokit/response/raise_error.rb:16:in `on_complete': POST https://api.github.com/repos/product-definition-center/pdc-ruby-gem/pulls/35/comments: 404 - Not Found // See: https://developer.github.com/v3/pulls/comments/#create-a-comment (Octokit::NotFound)

Seems remove this file the command can work, not know why yet.